[2025-10-20T23:36:44.153+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T23:36:44.176+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.load_data_task manual__2025-10-20T23:35:58.452593+00:00 [queued]>
[2025-10-20T23:36:44.185+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.load_data_task manual__2025-10-20T23:35:58.452593+00:00 [queued]>
[2025-10-20T23:36:44.185+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-20T23:36:44.207+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_data_task> on 2025-10-20 23:35:58.452593+00:00
[2025-10-20T23:36:44.219+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=282) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-20T23:36:44.224+0000] {standard_task_runner.py:63} INFO - Started process 295 to run task
[2025-10-20T23:36:44.221+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1_TaskFlow_FileIO', 'load_data_task', 'manual__2025-10-20T23:35:58.452593+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmp4sf5_bcb']
[2025-10-20T23:36:44.226+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask load_data_task
[2025-10-20T23:36:44.318+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1_TaskFlow_FileIO.load_data_task manual__2025-10-20T23:35:58.452593+00:00 [running]> on host 0074cd054b76
[2025-10-20T23:36:44.444+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='Airflow_Lab1_TaskFlow_FileIO' AIRFLOW_CTX_TASK_ID='load_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T23:35:58.452593+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T23:35:58.452593+00:00'
[2025-10-20T23:36:44.446+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T23:36:44.447+0000] {airflow.py:55} INFO - Loading data (writes artifact and returns path)
[2025-10-20T23:36:44.619+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/dags/data/loaded.pkl
[2025-10-20T23:36:44.621+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T23:36:44.650+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1_TaskFlow_FileIO, task_id=load_data_task, run_id=manual__2025-10-20T23:35:58.452593+00:00, execution_date=20251020T233558, start_date=20251020T233644, end_date=20251020T233644
[2025-10-20T23:36:44.682+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T23:36:44.715+0000] {taskinstance.py:3503} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-10-20T23:36:44.721+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
